flink-batch-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/TableInputFormat.java

```diff
@@ -67,18 +66,23 @@
 	protected abstract T mapResultToTuple(Result r);
```

Comment:
Can we remove the confusing comment `"abstract methods allow for multiple table and scanners in the same job"` and add JavaDocs to all abstract methods that describe what is expected from their implementation?

A `TableInputFormat` instance should only scan a single table. In case more tables need to be read, each could be read with a separate `TableInputFormat` instance and the output of those could be unioned if needed.